src/components/Cautions/Cautions.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ *
+ * @flow
+ */
+import * as React from 'react'
+import { View, Image, Text } from 'react-native'
+import I18n from 'react-native-i18n'
+import PropTypes from 'prop-types'
+import styles from './CautionsStyles'
+
+const CautionItem = ({ text }) => (
+  <View style={styles.item}>
+    <Text style={styles.bullet}>•</Text>
+    <Text style={styles.text}>{text}</Text>
+  </View>
+)
+
+CautionItem.propTypes = {
+    text: PropTypes.string,
+}
+
+const Cautions = () => (
+  <View style={styles.container}>
+    <Image
+      style={styles.image}
+      source={require('../../images/alert.png')}
```

Comment:
We must not use `require` in our source code. 
Please import image from src/images